cmd/alias-export.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"github.com/minio/cli"
+	json "github.com/minio/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/pkg/v2/console"
+)
+
+var aliasExportCmd = cli.Command{
+	Name:            "export",
+	ShortName:       "e",
+	Usage:           "export configuration info to stdout",
+	Action:          mainAliasExport,
+	OnUsageError:    onUsageError,
+	Before:          setGlobalsFromContext,
+	Flags:           globalFlags,
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} ALIAS ./credentials.json
+
+  Credentials to be exported must be in the following JSON format:
+  
+  {
+    "url": "http://localhost:9000",
+    "accessKey": "YJ0RI0F4R5HWY38MD873",
+    "secretKey": "OHz5CT7xdMHiXnKZP0BmZ5P4G5UvWvVaxR8gljLG",
+    "api": "s3v4",
+    "path": "auto"
+  }
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Export the provided credentials.json file as 'myminio' to the config:
+     {{ .Prompt }} {{ .HelpName }} myminio/ ./credentials.json
+
+  2. Export the credentials through standard input as 'myminio' to the config:
+     {{ .Prompt }} cat credentials.json | {{ .HelpName }} myminio/
```

Comment:
This makes no sense to me. It looks like an import operation.

Reply:
Copy paste error, forgot to remove